src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(person => {
+      const menCenture = Math.ceil(person.died / 100) === century;
+
+      return menCenture && person.sex === 'm';
+    })
+    : people.filter(person => person.sex === 'm');
```

Comment:
Maybe it will be easier but on your choice
```suggestion
  const men = people.filter(person => {
      const isDiedPersonOnExpectedCentury = century
        ? Math.ceil(person.died / 100) === century
        : true;

      return isDiedPersonOnExpectedCentury && person.sex === 'm';
    });
```